### DISCUSSION TITLE
Is MongoDB(instead of MySQL) supported using the nuget package

### DISCUSSION THREAD

**Discussion by niks12243648** [quartznet/quartznet#934] [General]
I have a usecase to move my complete infra from MySQL to MongoDB. We were using this nuget package to use quartz schedulers from our C# code.
Now we need to move the backend DB of quartz scheduler to MongoDB(from MySQL).
Are there configuration changes from which we can achieve this ? Or do we need to use some other nuget package?

**Comment by lahma** [comment]
Quartz project doesn't have direct support for MongoDB, but you should examine community contributed packages, [like this one](https://www.nuget.org/packages/Quartz.Spi.MongoDbJobStore/).
